okta-hosted-login/README.md

```diff
@@ -32,7 +32,8 @@ Plug these values into the `mvn` commands used to start the application.
 cd okta-hosted-login
 mvn -Dokta.oauth2.issuer=https://{yourOktaDomain}/oauth2/default \
     -Dokta.oauth2.clientId={clientId} \
-    -Dokta.oauth2.clientSecret={clientSecret}
+    -Dokta.oauth2.clientSecret={clientSecret} \
+    -Dokta.oauth2.postLogoutRedirectUri={absoluteLogoutRedirectUri} # (optional) add this when you want users to be logged out of the app (session cleared) and redirected to a uri. 
```

Comment:
This should say something about either an SSO logout or the OAuth term RP-Initiated logout
(Without this option the session of the application will still be cleared).

Maybe something like:
> (optional) configure this property to enable SSO logout
or :
> (optional) configure this property to enable an RP-initiated logout

(my concern is most folks will no know what an RP initiated logout is, so SSO logout might be a better fit 🤔

Reply:
ok noted. Will update to `SSO logout`.

Reply:
done